docs/learn-governance.md

```diff
@@ -264,11 +267,13 @@ against - equate to a simple majority-carries system at 100% turnout.
 
 ## Council
 
+> [Video explainer on Council](https://www.youtube.com/watch?v=837Vv3gdRzI)
+
 To represent passive stakeholders, Polkadot introduces the idea of a "council". The council is an
 on-chain entity comprising a number of actors, each represented as an on-chain account. On Polkadot,
 the council currently consists of 13 members. This is expected to increase over the next few months
 to 24 seats. In general, the council will end up having a fixed number of seats. On Polkadot, this
-will be 24 seats while on Kusama it is 17 seats.
+will be 24 seats while on Kusama it is 19 seats.
```

Comment:
will these values be changing often? perhaps it might be a better idea to use the injected script?

Reply:
Nope, very very rarely, but ripe for injection later.